packages/utils/resolve-packages.ts

```diff
@@ -18,9 +18,8 @@ const spawnChild = require("./package-manager").spawnChild;
  * @returns {Promise} promise - Returns a promise to the installation
  */
 
-function processPromise(child) {
-	return new Promise(function(resolve, reject) {
-		//eslint-disable-line
+export function processPromise(child: IChildProcess): Promise<any> {
+	return new Promise((resolve: () => void, reject: () => void) => {
```

Comment:
` (_): void `

Reply:
```js
(resolve: (_: void)
```

If we're doing this^ then what should we pass in `resolve()`?
We can do `(resolve: (_?: void)`, but looks ugly.

Reply:
Yeah just leave it without any types, it's better as Emanuel mentioned earlier this week

Reply:
Do you mean we should get rid of `_` argument everywhere or only this particular occurrence?

Reply:
Just for promises that resolve to void